lapack.go

```diff
@@ -48,8 +48,8 @@ type Float64 interface {
 type Direct byte
 
 const (
-	Forward  Direct = 'F' // Reflectors are right-multiplied, H_1 * H_2 * ... * H_k
-	Backward Direct = 'B' // Reflectors are left-multiplied, H_k * ... * H_2 * H_1
+	Forward  Direct = 'F' // Reflectors are right-multiplied, H(0) * H(1) * ... * H(k).
```

Comment:
Done, although here without context the final (or initial) subscript does not really matter.